api/src/main/resources/workbench.yaml

```diff
@@ -42,6 +42,25 @@ paths:
           schema:
             $ref: "#/definitions/Profile"
 
+  /api/v1/sendBug:
+    post:
+      tags:
+        - bugs
+      description: Sends an email to developers about a user reported bug
+      operationId: "sendBug"
```

Comment:
What's the difference between quoting or not quoting in these?

Reply:
I don't know if there is one, but it seems like the quoting has been our convention for all but one of the other ones. (getBillingProjects)
@danqrodney, do you know?

Reply:
I don't know. I have no preference here.

Reply:
If there isn't a difference, I'd prefer to type fewer characters.